File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/OrderInfoFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.payments.OrderInfo;

public class OrderInfoFilterValidator extends FilterValidator<OrderInfo> {

    public OrderInfoFilterValidator(CustomValidator<OrderInfo> validator) {
        super(OrderInfo.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/OrderInfoFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.payments.OrderInfo
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class OrderInfoFilterValidator(validator: Validator<OrderInfo>) : FilterValidator<OrderInfo>(
    OrderInfo::class, validator)
